02_Statistics/README.md

```diff
@@ -50,22 +50,188 @@ The median is the value __separating the higher half of a data sample, a populat
 
 ## 3_ Exploratory data analysis
 
+The step includes visualization and analysis of data. 
+
+Raw data may possess improper distributions of data which may lead to issues moving forward.
+
+Again, during applications we must also know the distribution of data, for instance, the fact whether the data is linear or spirally distributed.
+
+[Guide to EDA in Python](https://towardsdatascience.com/data-preprocessing-and-interpreting-results-the-heart-of-machine-learning-part-1-eda-49ce99e36655)
+
+##### Libraries in Python 
+
+[Matplotlib](https://matplotlib.org/)
+
+Library used to plot graphs in Python
+
+__Installation__:
+
+    sudo pip3 install matplotlib
+
+__Utilization__:
+
+    import matplotlib.pyplot as plt
+
+[Pandas](https://pandas.pydata.org/)
+
+Library used to large datasets in python
+
+__Installation__:
+
+    sudo pip3 install pandas
```

Comment:
No installation with sudo. Better to create a venv first.

Reply:
Yeah!! I missed that. Using "sudo" will install to the root. It's better to use a venv. Thanks